tests/Neo.UnitTests/IO/UT_ByteArrayComparer.cs

```diff
@@ -12,6 +12,7 @@
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Neo.IO;
+using System;
 
 namespace Neo.UnitTests.IO
```

Comment:
We should create a test project to `Neo.IO` and move these test to that project

Reply:
I planing on it.

Reply:
Why not in the same PR?

Reply:
This PR moves existing code to new library. There is or should be tests already created. This PR also doesn't change anything for application besides, now having an extra `dll`. In the next few PRs will be moving and checking for tests, if none than iIl create them. Just want to get the library projects started and not be to big.

@shargon